### ISSUE TITLE
gdm3/specterwm login crashes when trying to name a workspace not within the workspace_limit

### ISSUE THREAD

**Issue by lpanebr** [conformal/spectrwm#299]
I installed it with `sudo apt-get install spectrwm`

Restarted and logged in selecting **spectrwm** from the gdm login and it worked fine.

Then I never managed to log using spectrwm again.. I get sent back to the login screen after I enter my password... 

If I select ubuntu I'm able do log fine, but I really want to use spectrwm...

I already did a `sudo dpkg-reconfigure gdm3` but still the same..

Any ideas?

**Comment by LordReg** [comment]
The default *bar_font* setting uses an Xlib font set which may be failing to load.  spectrwm will not start if it cant load a font.  I suggest setting *bar_font* in your `spectrwm.conf` to use an Xft font.
For example:
```
bar_font = xos4 Terminus:pixelsize=14:antialias=true
```

**Comment by lpanebr** [comment]
I spent the night messing with it and found out what was causing the problem.

I heard on a YT video that spectrwm had very sane defaults and got confident to try just uncommenting stuff from the defaut config file.

However when I uncommented the lines below I undestood that **gdm3/specterwm crashes when trying to name a workspace not within the workspace_limit**.

This can be found in `/var/log/syslog`:
```sh
[lpanebr@neuron ~]$  grep workspace /var/log/syslog 
Apr 16 00:56:47 neuron /usr/lib/gdm3/gdm-x-session[2152]: spectrwm: setconfvalue: workspace_name: invalid workspace  10. 
```

**Conflicting sample default configuration:**
```bash
workspace_limit	= 9
...
# Set workspace name at start
name			= ws[1]:IRC
name			= ws[2]:Email
name			= ws[3]:Browse
name			= ws[10]:Music
```

**Comment by marcopeereboom** [comment]
@LordReg this needs to be looked at. Silly bug.

**Comment by LordReg** [comment]
The example spectrwm.conf actually has `# workspace_limit = 22`.  Leaving it commented would leave the default value of 10.
spectrwm generally quits with an error message (to stderr) when encountering a bad/conflicting user conf value.

In the case of setting workspace names, I see no issue continuing to load with a startup exception.
@marcopeereboom what do you think?

**Comment by marcopeereboom** [comment]
Sounds good to me.

**Comment by marcopeereboom** [comment]
Has this been fixed @LordReg?

**Comment by LordReg** [comment]
Yes, as part of #323 
Didn't merge that yet since I want to make sure you agree with not exiting on bad conf values.

**Comment by LordReg** [comment]
Merged #323
